manageiq-providers-amazon.gemspec

```diff
@@ -13,5 +13,8 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config.lib}/**/*"]
 
+  s.add_development_dependency "codeclimate-test-reporter", "~> 1.0.0"
+  s.add_development_dependency "simplecov"
```

Comment:
I've been putting dev dependencies *after* the runtime dependencies in other gems, so for consistency we may want to do that here as well.